### ISSUE TITLE
Make Alt+F4 work and warn the user about terminating more than one tab

### ISSUE THREAD

**Issue by arnederuwe** [microsoft/terminal#1589]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18362.175]
Windows Terminal version (if applicable): 0.2.1715.0
```

# Steps to reproduce

Start The terminal app, try to close it using Alt + F4

# Expected behavior

The app should close

# Actual behavior

The app doesn't close, which is inconsistent with 99% of windows apps

**Comment by Shorotshishir** [comment]
**On Second Thought.**
PowerShell and WSL individually don't close on `alt+f4` as they don't have it set for the task. Windows Terminal shows it has `alt+f4` set to close but it doesn't work.  perhaps they are trying to make it consistent with the rest two.

**Comment by DHowett-MSFT** [comment]
This is now the tracking issue for getting alt+f4 working and adding a dialog to stop users from harming their running session state.

**Comment by praneetloke** [comment]
It would also be nice if there was an option to prompt the user about closing a tab that has a running program. For example, running a development server of some kind in a tab in interactive mode (vs. daemon mode) is often a development workflow for all sorts of apps (Angular CLI, webpack-based apps, Node apps etc.) these days.

**Comment by DHowett-MSFT** [comment]
This was fixed by #2526.

**Comment by codenotworking** [comment]
I don't think this was fixed, because [#17613](https://github.com/microsoft/terminal/issues/17613) exists.

**Comment by DHowett** [comment]
@codenotworking thanks, we’ll keep all further discussion over there rather than on this *five year old* issue.
